floater/test/test_utils.py

```diff
@@ -95,24 +95,37 @@ def test_floats_to_netcdf(tmpdir, mitgcm_float_datadir_csv):
 
     input_dir = str(mitgcm_float_datadir_csv) + '/'
     output_dir = str(tmpdir) + '/'
-
     os.chdir(input_dir)
+
+    # least options
+    utils.floats_to_netcdf(input_dir=input_dir, output_fname='test')
+    # most options
```

Comment:
Don't you want to test the results of this first call to `floats_to_netcdf`? As far as I can see, you create the netcdf files, but then you immediately overwrite them in the next line.